src/assets/scripts/client/aircraft/FlightManagementSystem/WaypointModel.js

```diff
@@ -295,8 +339,10 @@ export default class WaypointModel {
         this._positionModel = null;
 
         this.isHold = false;
-        this.speedRestriction = -1;
-        this.altitudeRestriction = -1;
+        this.speedMaximum = -1;
```

Comment:
Added to global constants, and implemented everywhere `-1` is used like this, except to leave `indexOf() !== -1` as literal `-1`.